src/cli/parse_args.rs

```diff
@@ -0,0 +1,203 @@
+use super::{Arg, ArgDef};
```

Comment:
Ditto here. No way to (ab-)use `clap` for that? I can't escape the feeling we are inventing something existing here.